requirements.txt

```diff
@@ -1,5 +1,6 @@
 descartes>=1.1
 geopandas>=0.4
+lxml>=4.2
```

Comment:
is it necessary to add `lxml` as a dependency? is there a way to add this functionality (reasonably efficiently) using the python standard library?

Reply:
Done.

Reply:
Done.